pages/items.js.php

```diff
@@ -5463,15 +5463,28 @@ function(data) {
                 // Show restrictions with Badges
                 var html_restrictions = '';
                 $.each(store.get('teampassItem').id_restricted_to, function(i, value) {
-                    html_restrictions +=
+                    html_restrictions +=https://github.com/rcruz-pntlb/TeamPass/edit/bugfix/4422-fix-item-edit-after-csv-import/pages/items.js.php
                         '<span class="badge badge-info mr-2 mb-1"><i class="fa-solid fa-group fa-sm mr-1"></i>' +
                         data.users_list.find(x => x.id === parseInt(value)).name + '</span>';
                 });
+                /*BEGIN modification for issue #4422 bugfix/4422-fix-item-edit-after-csv-import
+                  original post: https://github.com/nilsteampassnet/TeamPass/issues/4422
+                    @Author:     rcruz@pintaluba.com
+                    @Date:       10/29/2024
+
+                --REPLACES ORIGINAL CODE--
                 $.each(store.get('teampassItem').id_restricted_to_roles, function(i, value) {
                     html_restrictions +=
                         '<span class="badge badge-info mr-2 mb-1"><i class="fa-solid fa-group fa-sm mr-1"></i>' +
                         data.roles_list.find(x => x.id === parseInt(value)).title + '</span>';
+                });                                                
+                */
```

Comment:
Can you remove the old code instead of comment it please?

Reply:
Hi @corentin-soriano 
ok!! ... and the rest of comments, better to remove, too?

Reply:
Comments should describe the associated code. Dates/issue links and others informations are in the git history!